config/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/SIGBlockchain/project_aurum/internal/config"
+)
+
+func main() {
+	//open configuration file
+	configFile, err := os.Open("../cmd/config.json")
+	if err != nil {
+		log.Fatal("Failed to load configuration file : " + err.Error())
+	}
+	defer configFile.Close()
+
+	cfgData, err := ioutil.ReadAll(configFile)
+	if err != nil {
+		log.Fatal("Failed to read configuration file : " + err.Error())
+	}
+
+	cfg := config.Config{}
+	if err := json.Unmarshal(cfgData, &cfg); err != nil {
```

Comment:
I believe we have a function that does the configuration loading. I believe it's in `internal/config/config.go`.

Reply:
The problem with that function is that it doesn't have the correct path to the `json` file, since this config file is in another directory.

Reply:
How about the `jsonify` package? It doesn't have a test to support it yet but maybe that would work?